insights/parsers/numeric_user_group_name.py

```diff
@@ -0,0 +1,62 @@
+"""
+NumericUserGroupName - command ``/usr/bin/grep -c '^[[:digit:]]' /etc/passwd /etc/group``
+=========================================================================================
+
+This parser reads the output of ``/usr/bin/grep -c '^[[:digit:]]' /etc/passwd /etc/group``
+and detects whether it is 0 or nonzero.
+
+The grep command returns the number of matches. It matches for user and group names that
+begin with a number. If it returns 0, then there are no such names, otherwise it returns
+the number of matches (names). The parser only detects whether it is 0 or nonzero so as
+to minimize the amount of collected data.
+
+The boolean is available in the ``data`` attribute but it should not be accessed directly
+(encapsulation). The parser provides methods for comparisons with bool and casting to bool.
+
+Sample input::
+
+    3
+
+Examples:
+
+    >>> bool(shared[NumericUserGroupName])
+    True
+    >>> True == shared[NumericUserGroupName]
+    True
+"""
+
+from .. import Parser, parser
+
+
+@parser('numeric_user_group_name')
+class NumericUserGroupName(Parser):
+    """
+    Reports whether there is a user or group with a name that starts with a digit.
+    """
+    def parse_content(self, content):
+        self.data = not ("0" == ''.join(content).strip())
```

Comment:
It might be simpler for users of this class, if you renamed this attribute to something like `has_numeric_user_group` so that they could just access the attribute directly.